llvm-spirv/test/extensions/INTEL/SPV_INTEL_fpga_reg/IntelFPGAReg.ll

```diff
@@ -368,7 +368,7 @@ attributes #4 = { nounwind }
 !11 = !{i64 0, i64 4, !9, i64 4, i64 4, !12}
 !12 = !{!13, !13, i64 0}
 !13 = !{!"float", !7, i64 0}
-!14 = !{i64 0, i64 4, !9, i64 0, i64 4, !15}
+!14 = !{i64 0, i64 4, !15}
```

Comment:
Do we know why this started failing? I will also check

Reply:
This is exposed by the newly enabled TBAA verfier.
* df75183d70e0 - (2024-03-27) [TBAA] Add verifier for tbaa.struct metadata (#86709)